### ISSUE TITLE
TypeError [ERR_INVALID_CALLBACK]: Callback must be a function

### ISSUE THREAD

**Issue by heyMP** [elmsln/wcfactory#148]
When running `wcf factory` node 8 throws the following warning:

```
(node:20541) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.
```

Node 10 throws this error:

```
events.js:167
      throw er; // Unhandled 'error' event
      ^

TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
    at makeCallback (fs.js:137:11)
    at Object.rename (fs.js:571:14)
    at module.exports.install (/Users/michael/.config/yarn/global/node_modules/@wcfactory/generator-wcfactory/generators/factory/index.js:54:8)
    at Object.<anonymous> (/Users/michael/.config/yarn/global/node_modules/yeoman-generator/lib/index.js:424:27)
    at /Users/michael/.config/yarn/global/node_modules/run-async/index.js:25:25
    at new Promise (<anonymous>)
    at /Users/michael/.config/yarn/global/node_modules/run-async/index.js:24:19
    at self.env.runLoop.add.completed (/Users/michael/.config/yarn/global/node_modules/yeoman-generator/lib/index.js:425:13)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
Emitted 'error' event at:
    at Immediate.setImmediate (/Users/michael/.config/yarn/global/node_modules/yeoman-generator/lib/index.js:433:22)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
```

**Comment by btopro** [comment]
`renameSync` is the synchronous successor to what used to be allowed in `rename` which in node 10+ requires a callback. Deprecation notice fixed in node 8 so I'm assuming this fixes it here.

I just published 0.4.1 which fixes this and another small issue; @azinck can you try again? `yarn global add @wcfactory/cli`

**Comment by azinck** [comment]
I can confirm this solves the problem for me @btopro -- thanks!
